tools/mkdocs_macros_bids/macros.py

```diff
@@ -196,6 +196,35 @@ def make_metadata_table(field_info, src_path=None):
     return table
 
 
+def make_sidecar_table(table_name, src_path=None):
+    """Generate a markdown table of metadata field information.
+
+    Parameters
+    ----------
+    field_names : dict
```

Comment:
table_name or field_names? sting or list of stings instead of dict.

Reply:
Oh, I probably didn't update this. Thanks for pointing it out...